### PR TITLE
CORE: View a list of all articles

### PR DESCRIPTION
**Pull request by tonymm55**
List all articles on the Home Page and filter browse by topics of football, cooking and coding. Also added some basic flex display to satisfy the ticket requirement. The functionality will expand as I progress through the tickets. I will have to add Article cards as I'm currently displaying as a list which I'm not sure is correct.

### CODE REVIEW

File: api.jsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import axios from "axios";
+
+const ncNewsAPI = axios.create({
+  baseURL: `https://northcoders-news-ykvv.onrender.com/api`,
+});
+
+export const fetchArticles = async (topic) => {
+  try {
+    const response = await ncNewsAPI.get(`/articles?topic=${topic}`);
+    console.log("POST request successful, response:", response);
```
Comment (nc-john-hutcheon) [inline api.jsx:10]: Ensure you're removing console logs before pushing your code 👍

File: api.jsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import axios from "axios";
+
+const ncNewsAPI = axios.create({
+  baseURL: `https://northcoders-news-ykvv.onrender.com/api`,
+});
+
+export const fetchArticles = async (topic) => {
+  try {
+    const response = await ncNewsAPI.get(`/articles?topic=${topic}`);
```
Comment (nc-john-hutcheon) [inline api.jsx:9]: This ticket was only meant to be to bring all the articles, without any option for filtering by topic at this point. You've actually done Ticket 10 as well here!

File: api.jsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import axios from "axios";
+
+const ncNewsAPI = axios.create({
+  baseURL: `https://northcoders-news-ykvv.onrender.com/api`,
```
Comment (nc-john-hutcheon) [inline api.jsx:4]: Nice work 👍

**Review by nc-john-hutcheon — COMMENTED** [review]
Hey Anthony! Good work. Please see my comments and just ensure in future you're only doing the work that the ticket describes! You've done Ticket 10 as well as 4 here! 😄

File: api.jsx (hunk 1 above)
Comment (tonymm55) [reply api.jsx:10]: Will do sorry - one day, that instruction will stick!!

File: api.jsx (hunk 2 above)
Comment (tonymm55) [reply api.jsx:9]: Whoops, apologies! I must look ahead too...

File: api.jsx (hunk 3 above)
Comment (tonymm55) [reply api.jsx:4]: Thanks and I will heed those comments...Good luck getting through all these Pull Requests!!! 🥵

**Pull request merged**